src/wallet/rpcwallet.cpp

```diff
@@ -1304,7 +1304,7 @@ UniValue listreceivedbyaddress(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() > 4)
         throw std::runtime_error(
             "listreceivedbyaddress ( minconf addlockconf include_empty include_watchonly)\n"
-            "\nList balances by receiving address.\n"
+            "\nList recieves by receiving address.\n"
```

Comment:
sp. - receives

Reply:
ohhh 🙈

Reply:
Actually, did this need to be changed? It does list the balance by address, right?

Reply:
I agree, I'm not sure "List receives by ..." makes sense (at least not to me).

Reply:
I think this matches what it does. What do you guys think?
"List transactions received by an address."

Reply:
Yep, basically these are not balances strictly saying (outgoing txes do not decrease the value).
"List transactions received by an address." is better but it's still not only txes, it's smth more... "incoming payments" maybe?

Reply:
`List (tx outputs|payments) to a receiving address`?

Reply:
So it is really an tx (in or out) associated with an address?

Reply:
Result looks like this btw:
```
[
  {
    "address": "yZCQYxJApDJCZ1FqGZsdU1jxvrpmLChVho",
    "account": "",
    "amount": 3.02987696,
    "confirmations": 1,
    "label": "",
    "txids": [
      "dc415b5814ed76762042dc89a640332e23bbb7988808c9de9f22c33582f8662a", 
      "4743b1984104f7627a02feac5558fa16cca989b9dd2365667710c9b64dc3103b", 
      "98a2b20cb8114b5d1ccd56afa78e0f8a2ec53a033f0c765e795a1fa57dbe1b61", 
      "2420d6bd07302cc25fba70dab8461c7a82b673a6556c13ca3db2f3deb20de666", 
      "4a93f439fb75ec0de2c979358f2fba56b42fc8c88e42609032d7b1b53c758a72", 
      "323fba12f4bab9e3349572a5f97d1b5f45c1a8dc20c6f340697f957468eeb8a1", 
      "80ecd6edfc12a12d5937ffed02d95ca196804d54ca3494cf7a0a850685c3b3ec", 
      "9e88f4ffcee5abc6ce3b7d2c3848a84c790c09600931a311887b3e5f4e0533f2"
    ]
  }
]
```
Only _incoming_ txes are included and counted.

Reply:
Got confused because I was looking at one that sent to itself :see_no_evil:

Reply:
Wait, so it's not specifying the tx outpoint, meaning if you get this list, then it's up to you to query the transaction and determine which output(s) paid to the address?

I'm also curious how `confirmations` is used here... is this the number of confirmations of the _latest_ transaction with some number of outputs which pays out to this address?

Reply:
@nmarley Yes, that sounds correct. And the confirmation count is for the latest tx. I finally looked it up on [this super helpful site](https://dash-docs.github.io/en/developer-reference#listreceivedbyaddress) :laughing:

Reply:
Ok, well whatever sounds good to you then. Seems like a funny call which only points in the right direction instead of serving fully useful info, but that's another issue entirely.

Reply:
Maybe `List incoming payments grouped by receiving address` and `List incoming payments grouped by account`? Should be clear enough.